open-metadata-implementation/governance-servers/integration-daemon-services/integration-daemon-services-registration/src/main/java/org/odpi/openmetadata/governanceservers/integrationdaemonservices/registration/IntegrationServiceDescription.java

```diff
@@ -100,6 +100,16 @@
                             "https://egeria.odpi.org/open-metadata-implementation/integration-services/analytics-integrator/",
                             "Analytics Modeling OMAS",
                             PermittedSynchronization.BOTH_DIRECTIONS),
+
+
+    SEARCH_INTEGRATOR_OMIS(4010,
+            "Search Integrator",
+            "Search Integrator OMIS",
+            "search-integrator",
+            "Integrator for a robust search engine",
```

Comment:
What does robust mean here? If we are going to use this term  and it is useful we should have it documented in our concepts.